doc/introduction/circuits.rst

```diff
@@ -14,15 +14,14 @@ Quantum circuits
     :target: javascript:void(0);
 
 
-In PennyLane, quantum computations are represented as *quantum node* objects. A quantum node is used to
+In PennyLane, quantum computations are represented as *quantum node* objects. A quantum node or QNode is used to
 declare the quantum circuit, and also ties the computation to a specific device that executes it.
-Quantum nodes can be easily created by using the :ref:`qnode <intro_vcirc_decorator>` decorator.
 
 QNodes can interface with any of the supported numerical and machine learning libraries---:doc:`NumPy <interfaces/numpy>`,
 :doc:`PyTorch <interfaces/torch>`, :doc:`TensorFlow <interfaces/tf>`, and
-`JAX <https://github.com/google/jax>`__---indicated by providing an optional ``interface`` argument
+:doc:`JAX <interfaces/jax>`---indicated by providing an optional ``interface`` argument
 when creating a QNode. Each interface allows the quantum circuit to integrate seamlessly with
-library-specific data structures (e.g., NumPy arrays, or Pytorch/TensorFlow tensors) and
+library-specific data structures (e.g., NumPy and JAX arrays or Pytorch/TensorFlow tensors) and
 :doc:`optimizers <optimizers>`.
 
 By default, QNodes use the NumPy interface. The other PennyLane interfaces are
```

Comment:
it strikes me that nowhere in the intro mentions 'circuits'. I feel like we should, since a new reader that clicks on this page likely wants to know how this page relates to circuits?

Reply:
I realize though that this is an existing problem with this page, separate to this PR!

Reply:
Ah, the connection between circuits <--> qfuncs is a bit below, in the Quantum Functions section 🙂

Reply:
> In PennyLane, quantum computations, which involve the execution of one or more quantum circuits, 
are represented as *quantum node* objects.

What about this opener?